lagunita/lms/static/sass/partials/base/_overrides.scss

```diff
@@ -17,6 +17,10 @@ $footer-logo-width: 100px;
 
 @import 'custom_overrides/_typography.scss';
 
+#register-extra {
```

Comment:
Can we get a before/after screenshot for this to see what is affected?

Reply:
Before:
![screen shot 2018-07-12 at 5 00 31 pm](https://user-images.githubusercontent.com/3364609/42665588-20c62610-85f5-11e8-8894-79465e997d5a.png)

After:
![screen shot 2018-07-12 at 2 33 35 pm](https://user-images.githubusercontent.com/3364609/42665595-260534f4-85f5-11e8-8bc9-334cc2d63405.png)